web/src/components/top.js

```diff
@@ -34,7 +34,7 @@ class TopEventsTable extends React.Component {
                 <tbody>
                   {this.props.data.map((x, index) =>
                     <tr key={index}>
-                      <td>{index}</td>
+                      <td>{index + 1}</td>
```

Comment:
This change could be done in another PR imo.

Reply:
ok

Reply:
done